### PR TITLE
Add optional logging to jlama requests and responses

### PR DESCRIPTION
**Pull request by mariofusco**
@geoand please give a look

### CODE REVIEW

**Review by geoand — APPROVED** [review]
LGTM, thanks!

**Comment by geoand** [comment]
CI is complaining about:

```
Error:  Failed to execute goal net.revelc.code:impsort-maven-plugin:1.12.0:check (check-imports) on project quarkus-langchain4j-jlama: Imports are not sorted in /home/runner/work/quarkus-langchain4j/quarkus-langchain4j/model-providers/jlama/runtime/src/main/java/io/quarkiverse/langchain4j/jlama/JlamaChatModel.java -> [Help 1]
```

File: model-providers/jlama/runtime/src/main/java/io/quarkiverse/langchain4j/jlama/JlamaChatModel.java [hunk 1]
```diff
@@ -8,12 +8,14 @@
 import java.util.Optional;
 import java.util.UUID;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```
Comment (geoand) [inline model-providers/jlama/runtime/src/main/java/io/quarkiverse/langchain4j/jlama/JlamaChatModel.java:12]: We usually use the JBoss Logging version

**Comment by mariofusco** [comment]
@geoand This should be fixed, please give a second look.

**Comment by mariofusco** [comment]
@geoand sorry wait :) I just saw your comment about the logging library.

**Comment by mariofusco** [comment]
@geoand ok, fixed now.

File: model-providers/jlama/runtime/src/main/java/io/quarkiverse/langchain4j/jlama/JlamaChatModel.java (hunk 1 above)
Comment (mariofusco) [reply model-providers/jlama/runtime/src/main/java/io/quarkiverse/langchain4j/jlama/JlamaChatModel.java]: Fixed.

**Review by geoand — APPROVED** [review]
Thanks!

**Pull request merged**